apps/src/codebridge/ControlButtons/index.tsx

```diff
@@ -145,6 +181,12 @@ const ControlButtons: React.FunctionComponent = () => {
         />
       ) : (
         <span className={moduleStyles.centerButton}>
+          {runTestTooltip &&
```

Comment:
This will only display the tooltip if you hover over the spinner and not the disabled buttons, right? Would making the tooltip wrap a div that contains both be worthwhile? I don't think you need to, given that the UI will probably change anyway, but we might want to update this later (so maybe a comment?).

I wonder if a "WithOptionalTooltip" component could simplify the logic in this (and maybe other places?) a little bit, something like...
```
const WithOptionalTooltip: React.FC<{
  shouldWrap: boolean;
  children: React.ReactNode;
  ...
}> = ({shouldWrap, children, ...}) => {
  if (shouldWrap) {
    return (
      <WithTooltip tooltipProps={...} >
        {children}
      </WithTooltip>
    );
  }
  return <>{children}</>;
};
```

But I see a `showTooltip` prop in `WithTooltip` so maybe that's confusing or we should be leveraging that somehow but either way, then we're definitely outside the scope of this PR 😅

Reply:
I'll add a comment! In `WithTooltip` I see a `showTooltip` state, not prop, and I think it represents if the tooltip is currently being moused over, so I didn't think I could utilize that to make the tooltip optional. I like the idea of `WithOptionalTooltip`, if I mess with this in the future I'll add something!